src/swagger.yml

```diff
@@ -2963,6 +3290,230 @@ definitions:
+        type: string
+        description: Person's firstname
+      lastname:
+        type: string
+        description: Person's lastname
+      address:
+        type: string
+        description: Person's address
+      zip_code:
+        type: integer
+        format: int32
+        minimum: 1000
+        maximum: 9699
+        description: Person city's ZIP code
+      city:
+        type: string
+        description: Person's city
+      email:
+        type: string
+#        pattern: @todo Fixed pattern?
+        description: Person's (public) e-mail address
+      gender:
+        type: string
+#        pattern: '/^(male|female)$/'
+        enum:
+          - female
+          - male
+        description: Person's gender (male or female)
+      shop_user_id:
+        type: integer
+        format: int32
+        description: Person's shop ID
+      language:
+        type: string
+        description: Person's locale
+      article:
+        $ref: '#/definitions/AppraisalArticle'
+      marked:
+        type: boolean
+        description: Is appraisal marked?
+      processed:
+        type: boolean
+        description: Is appraisal processed?
+      approved:
+        type: boolean
+        description: Is appraisal approved?
+      approved_image:
+        type: boolean
+        description: Is image approved?
+      source:
+        type: string
+#        pattern: '/^(web|mobile)$/'
+        description: Where the appraisal originated from ("web" or "mobile")
+      type:
+        type: string
+#        pattern: '/^(article|wine_selfie)$/'
+        description: Type of appraisal ("article" or "wine_selfie")
+    required:
+      - id
+      - title
+      - rating
+      - article
+
+  AppraisalArticle:
+    type: object
+    properties:
+      id:
+        type: string
+        pattern: '/^\d{6}$/'
+        description: Navision article code
+      name:
+        type: string
+        description: Article name
+    required:
+      - id
+
+  AppraisalStatistic:
+    type: object
+    properties:
+      total:
+        type: integer
+        format: int32
+        description: Total appraisals in the range
+      processed:
+        type: integer
+        format: int32
+        description: Appraisals in the range that have been processed
+      not_processed:
+        type: integer
+        format: int32
+        description: Appraisals in the range that have not been processed
+      approved:
+        type: integer
+        format: int32
+        description: Appraisals in the range that have been approved
+      not_approved:
+        type: integer
+        format: int32
+        description: Appraisals in the range that have not been approved
+      unique:
+        type: integer
+        format: int32
+#        description:
+      sweepstake_total:
+        type: integer
+        format: int32
+#        description:
+      sweepstake_unique:
+        type: integer
+        format: int32
+#        description:
```

Comment:
Total number of individual persons that participated at least once in the sweepstake (based on email addresses).